Assignment2/felipeleonfernandez-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,48 @@
+{
+	"@context": {
+		"vocab": "http://upmsemanticweb.com/students/#",
+	    "fullName": "vocab:fullName",
+		"hometown": "vocab:hometown",
+		"age": "vocab:age",
+		"almaMater": "vocab:almaMater",
+		"parentsName": "vocab:parentsName",
+		"father": "parentsName:father",
+		"mother": "parentsName:mother",
+		"subjects": "vocab:subjects",
+		"name": "subjects:name",
+		"teachers": "subjects:teachers",
+		"description": "subjects:description"
+	},
+	"@id": "http://upmsemanticweb.com/students/felipeleonfernandez",
+	"@type": "vocab:felipeleonfernandez",
```

Comment:
It is better if you define you as a vocab:Person or vocab:Student, only a minor comment.